src/clients/runtime.ts

```diff
@@ -0,0 +1,67 @@
+import { IOContext } from '@vtex/api'
+import { ManifestEditor } from '../lib/manifest'
+import { cluster } from '../env'
+import * as url from 'url'
+import * as WebSocket from 'ws'
+import { getToken } from '../conf'
+
+const EOT = '\x04'
+
+export class Runtime {
+  private region: string
+  private account: string
+  private workspace: string
+
+  constructor(context: IOContext) {
+    const { region, account, workspace } = context
+    this.region = region
+    this.account = account
+    this.workspace = workspace
+  }
+
+  public executeCommand = async (command: string, interactive: boolean) => {
```

Comment:
In this case you could use `public async executeCommand(...`
When using classes I had doubts on when using named functions `public async executeCommand() {` or arrow functions `public executeCommand = () => {`. The difference in terms of behavior is shown in this example:
```

class A {
  constructor(x) {
    this.x = x
  }

  named() {
    console.log(this.x)
  }

  arrow = () => {
    console.log(this.x)
  }
}

const testArrow = () => {
    const a = new A(300)
 
    // Will print 300
    a.arrow()
    
    const fn1 = a.arrow
    // Will print 300
    fn1()
}

const testNamed = () => {
    const a = new A(400)

    // Will print 400
    a.named()

    const fn1 = a.named
    // Doesn't work - when a.named is decouped from the object 'a' the this variable is decoupled as well, so this.x won't exist 
    fn1() 
}

testArrow()
testNamed()
```

That occurs because when an arrow function is created it binds the content inside it with the current `this` variable, preventing decoupling.

However, creating an arrow function is not so performatic as a named function on a class - so if `runtime.executeCommand` isn't decouple from its object (all calls to it are `runtime.executeCommand(...)`), it's better to use named functions

https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1

Of course in toolbelt we won't have performance issues, but I think maybe it's a good practive to prefer named functions in classes